test/token/ERC20/BasicToken.test.js

```diff
@@ -1,82 +0,0 @@
-const { assertRevert } = require('../../helpers/assertRevert');
-const BasicToken = artifacts.require('BasicTokenMock');
-
-contract('StandardToken', function ([_, owner, recipient, anotherAccount]) {
```

Comment:
We shouldn't delete the `BasicToken` tests, rather, they should be added to the `StandardToken` test suite.

Reply:
(don't be confused by the fact that this test was wrongly named `StandardToken`)

Reply:
@nventuro Thanks for the feedback! Now that `BasicToken` has been merged w/ `StandardToken`, I went through `BasicToken.test.js` and noticed the exact same units tests are already in `StandardToken.test.js`. 

Am I missing something?

Reply:
No, you're right, I didn't notice the tests were the same. Thanks for removing that code duplication :)